python/activator/middleware_interface.py

```diff
@@ -282,20 +278,55 @@ def _prep_collections(self):
         #             'refcats'],
         #     output=self.output_collection)
         # The below is taken from SimplePipelineExecutor.prep_butler.
-        output_run = f"{self.output_collection}/{self.instrument.makeCollectionTimestamp()}"
+        self.output_run = f"{self.output_collection}/{self.instrument.makeCollectionTimestamp()}"
         self.butler.registry.registerCollection(self.instrument.makeDefaultRawIngestRunName(),
                                                 CollectionType.RUN)
-        self.butler.registry.registerCollection(output_run, CollectionType.RUN)
+        self.butler.registry.registerCollection(self.output_run, CollectionType.RUN)
         self.butler.registry.registerCollection(self.output_collection, CollectionType.CHAINED)
-        collections = [self.calibration_collection,
+        collections = [self.instrument.makeCollectionName("defaults"),
+                       self.instrument.makeCalibrationCollectionName(),
                        self.instrument.makeDefaultRawIngestRunName(),
                        'refcats',
-                       output_run]
+                       self.output_run]
```

Comment:
The point of using a defaults collection would be that it already includes the calibration and refcats collections, and moves the responsibility for getting it right out of the code and into the dataset.
```suggestion
        collections = [self.instrument.makeCollectionName("defaults"),
                       self.instrument.makeDefaultRawIngestRunName(),
                       self.output_run]
```

Reply:
Ok, this works correctly now; it didn't while I was debugging, but I think I'm finally exporting enough chained collections.